Added the v3 endpoint for newsleters_subscriptions

santostiago opened the pull request:
This is a very simple feature, as it's just copying it from V2.

agnessa commented on config/routes.rb:
```diff
@@ -15,6 +15,7 @@
         resources :columns, only: [:index]
       end
       get '/get_all_nodes', to: 'nodes#get_all_nodes'
+      resources :newsletter_subscriptions, only: [:create]
```
maybe don't remove it from v2 just yet, so that we don't break anything in the frontend?

santostiago replied:
Yes, I know. I didn't remove it. The routes were duplicated, so I just removed one of them.
It still exists in line 24.

tiagojsag commented:
no tests??????

santostiago commented:
@tiagojsag I've done the tests, but I removed them as I found them pointless.
For the test to pass, there should be a valid mail chimp API key. And it would actually add the email provided to the mailing list (as this is the only thing this endpoint does). So I thought it would be better to remove the test.
@agnessa I've updated the swagger file.
